### DISCUSSION TITLE
is there any way to display table in tabs already existed in form?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#7850] [Help]
public function form(Form $form): Form
    {
        return $form
            ->schema([
Tabs::make('Label')
    ->tabs([
        Tabs\Tab::make('Tab 1')
            ->schema([
                // ...
            ]),
        Tabs\Tab::make('Tab 2')
            ->schema([
                // ...
            ]),
        Tabs\Tab::make('Tab 3')
            ->schema([
                // ...
            ]),
    ])
]),
}

**Comment by pboivin** [comment]
You can't currently embed a Table in a Form. This could change in the future.

**Reply by devhoussam1998** [reply]
oh unfortunately I need this feature

**Reply by pboivin** [reply]
If you're only trying to display information, you could do a custom HTML `<table>` inside of a form `View` component. You could even reuse some of the HTML and CSS classes from Filament to match the look of the Table Builder:
https://filamentphp.com/docs/3.x/forms/layout/custom#view-components
